aspnetcore-webapi/SerializationDemo/SerializationDemo.Client/Clients/CachedJsonClient.cs

```diff
@@ -0,0 +1,82 @@
+﻿using Microsoft.Extensions.Caching.Distributed;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.DependencyInjection;
+using SerializationDemo.Client.Serialization;
+using SerializationDemo.Client.Serialization.Json;
+using SerializationDemo.Common.Models;
+using System.Net.Http.Headers;
+
+namespace SerializationDemo.Client.Clients
+{
+    internal class CachedJsonClient : ClientBase, IClient
+    {        
+        private readonly IByteSerializer _serializer = new JsonByteSerializer();
+        private readonly HttpClient _client = new HttpClient();
+        private readonly IDistributedCache _distributedCache;
+        private const string AllEmployeesKey = "AllEmployees";
+        public CachedJsonClient()
```

Comment:
Newline between the fields and the constructor.

Reply:
done